### DISCUSSION TITLE
acceptMetamaskAccess() not working properly

### DISCUSSION THREAD

**Discussion by tonykha123** [Synthetixio/synpress#583] [General]
Hi there, I'm using Synpress for my dapp, and everything seems to be working properly besides the acceptMetamaskAccess(). It seems to not register that the task is done even though it has successfully been completed.
![image](https://user-images.githubusercontent.com/93644855/202312971-0eb3f24b-a285-4b44-9ced-452eb6a13013.png)

As you can see here, it has accepted the metamask access going onto the next step which is the signature request. However since the task didnt register as being completed it just times itself out and cannot proceed to the next step which is accepting the signature request.

![image](https://user-images.githubusercontent.com/93644855/202310900-7764fc72-9464-4ded-af05-5d82264e5d9e.png)

here is my code for the test case
![image](https://user-images.githubusercontent.com/93644855/202311021-b963157e-072a-42bc-bad9-5d54c6c00b0d.png)

**Comment by drptbl** [comment]
https://github.com/Synthetixio/synpress/issues/584#issuecomment-1317939587
